notebooks/00_quick_start/wide_deep_model_movielens.ipynb

```diff
@@ -0,0 +1,1388 @@
+{
```

Comment:
<p>the notation `_items` is  weird in a notebook. Adding `_` to a name typically means that this variable is private, but in a notebook there is no such thing.  I would recommend to rename `_items` to either `items` or a different name (without `_`)</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Changing